mtags/src/main/scala-2/scala/meta/internal/pc/completions/AmmoniteCompletions.scala

```diff
@@ -106,6 +108,72 @@ trait AmmoniteCompletions { this: MetalsGlobal =>
       }
   }
 
+  case class AmmoniteIvyCompletions(
+      select: Tree,
+      selector: List[ImportSelector],
+      pos: Position,
+      editRange: l.Range
+  ) extends CompletionPosition {
+
+    override def contribute: List[Member] =
+      try {
+
+        val query = selector.collectFirst {
+          case sel: ImportSelector if sel.name.toString().contains(CURSOR) =>
+            sel.name.decode.replace(CURSOR, "")
+        }
+
+        query match {
+          case Some(imp) =>
+            val api = coursierapi.Complete
+              .create()
+              .withScalaVersion("2.13.8")
+              .withScalaBinaryVersion("2.13")
+
+            def completions(s: String): List[String] =
+              api.withInput(s).complete().getCompletions().asScala.toList
+            val javaCompletions = completions(imp)
+            val scalaCompletions =
+              if (imp.endsWith(":") && imp.count(_ == ':') == 1)
+                completions(imp + ":").map(":" + _)
+              else List.empty
```

Comment:
```suggestion
              if (imp.endsWith(":") && !imp.endsWith("::") )
                completions(imp + ":").map(":" + _)
              else List.empty
```

Reply:
Hmmm I think the suggestion will prompt an extra api call at for example 

```scala
import $ivy.`org.scalameta::metals:|`
```
?

Reply:
ach ok then, I didn't realize it was also in that case. nvm!